lesson06/assignment/contact.html

```diff
@@ -0,0 +1,88 @@
+        <ul class="navbar-nav nav__ul list-inline">
+          <li class="nav-item nav__li list-inline-item"><a class="nav-link nav__link" href="index.html">Home</a></li>
+          <li class="nav-item nav__li list-inline-item"><a class="nav-link nav__link" href="attractions.html">Attractions</a></li>
+          <li class="nav-item nav__li display-3 list-inline-item nav__spacelandtitle d-none d-md-block">SPACELAND</li>
+          <li class="nav-item nav__li list-inline-item"><a class="nav-link nav__link" href="about.html">About</a></li>
+          <li class="nav-item active nav__li list-inline-item"><a class="nav-link nav__link" href="">Contact</a></li>
+        </ul>
+      </div>
+    </nav>
+    <!--desktop nav bar sub-title-->
+    <div class="main__h1 d-none d-md-block">
+      <h1 class="display-5">THE NOSTALGIA AND EXCITEMENT OF SPACE IN ONE PLACE</h1>
+    </div>
+
+    <h2 class="text-danger text-center m-4 p-5">CONTACT</h2>
+
+    <div class="m-5" id="accordion">
+      <div class="card m-5">
+        <div class="card-header" id="headingOne">
+          <h5 class="mb-0">
+            <button class="btn btn-link" data-toggle="collapse" data-target="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+              E-mail Us
+            </button>
+          </h5>
+        </div>
+
+        <div id="collapseOne" class="collapse" aria-labelledby="headingOne" data-parent="#accordion">
+          <div class="card-body text-danger text-center">
+            Feel free to send us an e-mail via spaceland@gmail.com
+          </div>
+        </div>
+      </div>
+      <div class="card m-5">
+        <div class="card-header" id="headingTwo">
+          <h5 class="mb-0">
+            <button class="btn btn-link collapsed" data-toggle="collapse" data-target="#collapseTwo" aria-expanded="false" aria-controls="collapseTwo">
+              Call Us
+            </button>
+          </h5>
+        </div>
+        <div id="collapseTwo" class="collapse" aria-labelledby="headingTwo" data-parent="#accordion">
+          <div class="card-body text-danger text-center">
+            Feel free to call us at 1 - 800 - SPACELAND
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <footer class="main__footer">
+      <p>SPACELAND &copy; 2020</p>
+    </footer>
+
+  </main>
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js" integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN" crossorigin="anonymous"></script>
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js" integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV" crossorigin="anonymous"></script>
+  <script src="scripts/script.js"></script>
+</body>
```

Comment:
Scripts should be loaded in the head element. The 'defer' attribute can be used to prevent external scripts from loading until all other content on the page has loaded, so that they don't cause any lag to loadtime (https://www.w3schools.com/tags/att_script_defer.asp). Custom javascript that depends on finding elements on the page should be wrapped in a DOM content loaded event block (https://developer.mozilla.org/en-US/docs/Web/API/Window/DOMContentLoaded_event). After this assignment, we'll be moving into frameworks, so it won't be an issue again, but please be aware of this standard in professional code.